BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/DataSource.java

```diff
@@ -0,0 +1,52 @@
+package com.izor066.android.blocspot.api;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.izor066.android.blocspot.BuildConfig;
+import com.izor066.android.blocspot.api.model.PointsOfInterest;
+import com.izor066.android.blocspot.api.model.database.DatabaseOpenHelper;
+import com.izor066.android.blocspot.api.model.database.table.PointsOfInterestTable;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class DataSource {
+
+    private DatabaseOpenHelper databaseOpenHelper;
+    private PointsOfInterestTable pointsOfInterestTable;
+
+
+
+    public DataSource (Context context) {
+        pointsOfInterestTable = new PointsOfInterestTable();
+        databaseOpenHelper = new DatabaseOpenHelper(context, pointsOfInterestTable);
+
+        if (BuildConfig.DEBUG && true) {
```

Comment:
As `BuildConfig.DEBUG` already returns a boolean, there is no need of asserting with an additional `&& true`. 
`if (BuildConfig.DEBUG)` alone is sufficient.